internal/driver/glfw/window.go

```diff
@@ -1107,10 +1095,10 @@ func (w *window) capturesTab(keyName fyne.KeyName, modifier desktop.Modifier) bo
 		if ent, ok := w.canvas.Focused().(fyne.Tabbable); ok && !ent.AcceptsTab() {
 			switch modifier {
 			case 0:
-				w.queueEvent(func() { w.canvas.FocusNext() })
+				w.QueueEvent(func() { w.canvas.FocusNext() })
```

Comment:
You can pass the function directly instead of wrapping it.

Reply:
You are right. I will update :)

Reply:
Done! :)